packages/integration-tests/index.js

```diff
@@ -355,20 +426,16 @@ async function listCollections(stackName, bucketName, dataDirectory) {
  * @returns {Promise.<number>} number of deleted collections
  */
 async function deleteCollections(stackName, bucketName, collections, postfix) {
-  setProcessEnvironment(stackName, bucketName);
-
-  const promises = collections.map((collection) => {
-    if (postfix) {
-      collection.name += postfix;
-      collection.dataType += postfix;
-    }
-    const c = new Collection();
-    const id = constructCollectionId(collection.name, collection.version);
-    console.log(`Deleting collection ${id}`);
-    return c.delete(collection);
-  });
+  await Promise.all(
+    collections.map(
+      ({ name, version }) => {
+        const realName = postfix ? `${name}${postfix}` : name;
+        return collectionsApi.deleteCollection(stackName, realName, version);
+      }
+    )
+  );
 
-  return Promise.all(promises).then((cs) => cs.length);
+  return collections.length;
```

Comment:
Same comment as above regarding the length of the result

Reply:
See https://github.com/nasa/cumulus/pull/1475/files#r379627630